Integrations/integration-Carbon_Black_Enterprise_Live_Response.yml

```diff
@@ -1325,13 +1325,13 @@ script:
         return commandEntries['create process'](command);
     }
 
-    function memdumpRequest(sessionId, path, commpress) {
+    function memdumpRequest(sessionId, path) {
         let queryParams = {
             name: 'memdump',
-            object: path
+            object: path,
         };
-        if (commpress) {
-            queryParams.commpress = commpress;
+        if (args.compress) {
+            queryParams.compress = true;
```

Comment:
I believe this change is incorrect as the CB API has it as `commpress`. Looking at the API documentation [here](https://developer.carbonblack.com/reference/enterprise-response/5.1/live-response-api/#memdump).

Reply:
Hi, the API documentation is incorrect, it looks like a typo.
Here is the result of running the updated command:
![image](https://user-images.githubusercontent.com/31487955/62886233-a6040e00-bcef-11e9-890d-59055602ea85.png)